### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Bysheim**
lez go

### CODE REVIEW

File: src/app/projects/lars/bingo/common/components/ProtectedRoute.tsx
Comment (X104n) [inline src/app/projects/lars/bingo/common/components/ProtectedRoute.tsx:1]: Please explain this to me one day :)

File: src/app/projects/lars/bingo/common/config/firebase.js
Comment (X104n) [inline src/app/projects/lars/bingo/common/config/firebase.js:1]: Open codes for the internett?! 👀

**Review by X104n — APPROVED** [review]
Looks good, cant look through it properly bc. its election day🕋

**Review by ArneeMe — APPROVED** [review]
Ser bra ut man

**Pull request merged**